proto/disaggregated.proto

```diff
@@ -43,3 +47,73 @@ message TryMarkDeleteRequest {
 message TryMarkDeleteResponse {
     S3LockResult result = 1;
 }
+
+/// For compute task dispatch and data exchange ///
+
+message DisaggregatedTaskMeta {
+    uint64 start_ts = 1; // start ts of a query
+    // query_ts + server_id + local_query_id to represent a global unique query.
+    uint64 query_ts = 2; // timestamp when start to execute query, used for TiFlash miniTSO schedule.
+    uint64 server_id = 3; // TiDB server id
+    uint64 local_query_id = 4; // unique local query_id if tidb don't restart.
+    int64 task_id = 5; // if task id is -1 , it indicates a tidb task.
+    string executor_id = 6; // the exectuor id
+}
+message DisaggReadError {
+	int32 code = 1;
+	string msg = 2;
+}
+
+message EstablishDisaggregatedTaskRequest {
+    DisaggregatedTaskMeta meta = 1;
```

Comment:
How about Let's unify the name as Disagg?

Reply:
done